benchmark/child_process/child-process-exec-stdout.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const common = require('../common.js');
+const bench = common.createBenchmark(main, {
+  len: [64, 256, 1024, 4096, 32768],
+  dur: [5]
+});
+
+const exec = require('child_process').exec;
+function main(conf) {
+  bench.start();
+
+  const dur = +conf.dur;
+  const len = +conf.len;
+
+  const msg = '"' + Array(len).join('.') + '"';
```

Comment:
But also run `msg.test(/./)` to flatten the string. Creates more reliable benchmark results.